modules/manage/pages/security/index.adoc

```diff
@@ -5,4 +5,10 @@
 :page-categories: Management, Security
 
 
-NOTE: All concepts described in this section are compatible with Kafka and its client libraries and CLIs. This section does not cover ways you can protect your Redpanda cluster externally; for example, through network ACLs or private networks.
+[NOTE]
+====
+All concepts described in this section are compatible with Kafka and its client libraries and CLIs. This section does not cover ways you can protect your Redpanda cluster externally; for example, through network ACLs or private networks.
```

Comment:
@kbatuigas I'm not sure this is accurate anymore technically since e.g. Schema Registry AuthZ isn't part of kafka and this whole section describes a bunch of stuff not part of kafka.   I think we should just leave the second sentence and skip the first one.